### ISSUE TITLE
New release for Swift 4.1

### ISSUE THREAD

**Issue by ilyapuchka** [krzysztofzablocki/Sourcery#566]

**Comment by kdawgwilk** [comment]
I have tried building my own `Sourcery.app` to use until the release is created and I am seeing some weird errors just so you are aware
```
Using configuration file at '.sourcery.yml'
Scanning sources...
[1]    21585 bus error
```

**Comment by ilyapuchka** [comment]
@krzysztofzablocki for now I discovered some breaking changes in parsing typealiases, attributes and subscripts (SourceKit output changed). Do you think it makes sense to make release with what we have already that still targets Swift 4 while we are working on Swift 4.1 support? I don't think that it will be easy to maintain both versions but also it can take some time to fix issues I discovered.

**Comment by ilyapuchka** [comment]
I would probably do release now and announce it as the last that will support Swift 4 and that the next version will be only fully compatible with Swift 4.1

**Comment by ilyapuchka** [comment]
Resolved with #571
